vividus-extension-selenium/src/main/java/org/vividus/ui/context/UiContext.java

```diff
@@ -39,32 +40,26 @@
     @Override
     public SearchContext getSearchContext()
     {
-        SearchContextData searchContext = testContext.get(KEY, SearchContextData.class);
-        if (searchContext != null)
-        {
-            return searchContext.getSearchContext();
-        }
-        return null;
+        return getSearchContext(SearchContext.class).orElse(null);
```

Comment:
Shouldn't we return Optional<SearchContext> here also?

Reply:
yes, we should, that will be the next step and it'll result in a storm of changes, the current PR is more POC